code/stable/dblpendulum/src/cpp/README.md

```diff
@@ -1,7 +1,7 @@
 # DblPendulum 
 > Author: Dong Chen
 
-> Purpose: efficiently and correctly predict the motion of a pendulum.
+> Purpose: predict the motion of a pendulum.
```

Comment:
Should "predict" be capitalized? (This applies to the other READMEs as well)

Reply:
I believe that the reason that they are not is because READMEs obtain the purpose from the `purp` variable in the examples. The `purp` has the `Sentence` not capitalized because it is attached to a boilerplate text that results in a sentence such as the following: 

![image](https://github.com/JacquesCarette/Drasil/assets/110783820/64c3d61e-7d8b-4867-9e02-b00d3ac9b5ad)

The boilerplate text is "A system is needed to".

Reply:
Is there a way to capitalize a `Sentence` after it's been created? If not, there should be an issue created for that (if there isn't one already; I know we've started investigating something similar!)

Reply:
I was under the impression that the location at which the READMEs take the input for the purpose was a `Sentence`. However, it is a `String` and can be capitalized using the `capitalize` function. I am making that change right now and will commit the changes soon! Thanks for catching that!